pkg/apis/scope/v0alpha1/register.go

```diff
@@ -37,18 +37,20 @@ var (
 )
 
 func init() {
-	localSchemeBuilder.Register(addKnownTypes)
+	localSchemeBuilder.Register(func(s *runtime.Scheme) error {
+		return AddKnownTypes(SchemeGroupVersion, s)
+	})
 }
 
 // Adds the list of known types to the given scheme.
-func addKnownTypes(scheme *runtime.Scheme) error {
-	scheme.AddKnownTypes(SchemeGroupVersion,
+func AddKnownTypes(gv schema.GroupVersion, scheme *runtime.Scheme) error {
+	scheme.AddKnownTypes(gv,
 		&Scope{},
 		&ScopeList{},
 		&ScopeDashboardBinding{},
 		&ScopeDashboardBindingList{},
 	)
-	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
```

Comment:
When we register the internal version, this fails with the error:
```
Double registration of different types for scope.grafana.app/__internal, Kind=WatchEvent: old=k8s.io/apimachinery/pkg/apis/meta/v1.InternalEvent, new=k8s.io/apimachinery/pkg/apis/meta/v1.WatchEvent in scheme "pkg/runtime/scheme.go:100
```

it does not appear to be used, so maybe we just remove it?